src/pages/styles.module.css

```
@@ -35,3 +37,36 @@
   height: 200px;
   width: 200px;
 }
+
+/**
+* For table of content items 
+*/
+.TOC {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-self: center;
+  margin: 1.5vh 0;
+  padding: 0;
+}
+
+.TOC > ul {
+  display: inline-block;
+  text-align: center;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
```

You can actually combine this nesting! It'll work for all of the other nesting pieces as well

```suggestion
.TOC {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-self: center;
  margin: 1.5vh 0;
  padding: 0;
  
  ul {
    display: inline-block;
    text-align: center;
    list-style: none;
    padding: 0;
    margin: 0;
  }
  
  /* ... The other nesting */
}
```